decidim-participatory_processes/lib/decidim/participatory_processes/engine.rb

```diff
@@ -61,6 +61,12 @@ class Engine < ::Rails::Engine
                     active: :inclusive
         end
       end
+
+      initializer "decidim_participatory_processes.view_hooks" do
+        Decidim.view_hooks.register(:highlighted_elements, priority: Decidim::ViewHooks::HIGH_PRIORITY) do |view_context|
+          view_context.render partial: "decidim/participatory_processes/pages/home/highlighted_processes"
```

Comment:
We should load `participatory_processes` here (extracted from the previous code).